WordPress/src/main/java/org/wordpress/android/ui/prefs/AppPrefs.java

```diff
@@ -959,4 +962,36 @@ public static void setSystemNotificationsEnabled(boolean enabled) {
     public static boolean getSystemNotificationsEnabled() {
         return getBoolean(UndeletablePrefKey.SYSTEM_NOTIFICATIONS_ENABLED, true);
     }
+
+    private static List<String> getPostWithHWAccelerationOff() {
+        String idsAsString = getString(DeletablePrefKey.AZTEC_EDITOR_DISABLE_HW_ACC_KEYS, "");
+        return Arrays.asList(idsAsString.split(","));
+    }
+
+    /*
+     * adds a local site ID to the top of list of recently chosen sites
+     */
+    public static void addPostWithHWAccelerationOff(int localSiteId, int localPostId) {
+        if (localSiteId == 0 || localPostId == 0 || isPostWithHWAccelerationOff(localSiteId, localPostId)) {
+            return;
+        }
+        String key = localSiteId + "-" + localPostId;
+        List<String> currentIds = getPostWithHWAccelerationOff();
```

Comment:
Let's make this line `List<String> currentIds = new ArrayList<String>(getPostWithHWAccelerationOff());`, given `getPostWithHWAccelerationOff()` returns an `Arrays.asList()` which is a "fixed-size list" as per [the docs](https://docs.oracle.com/javase/7/docs/api/java/util/Arrays.html#asList(T...)), and thus the following line of code 
`currentIds.add(key);` will crash otherwise

Reply:
Resolved in 6f280e46dd2d5c93c7c2424fed0cac6d764f174e

Reply:
🤕